encrypted-media/scripts/onencrypted.js

```diff
@@ -0,0 +1,49 @@
+function runTest(config) {
+    var expectedInitData = [];
+    expectedInitData.push(stringToUint8Array(atob(config.keys[0].initData)));
+    expectedInitData.push(stringToUint8Array(atob(config.keys[1].initData)));
+
+    // Will get 2 identical events, one for audio, one for video.
+    var expectedEvents = 2;
+    var currentData;
+
+    async_test(function (test) {
+        var video = config.video,
+          mediaSource,
+          onEncrypted = function (event) {
+              currentData = new Uint8Array(event.initData);
+              assert_equals(event.target, config.video);
+              assert_true(event instanceof window.MediaEncryptedEvent);
+              assert_equals(event.type, 'encrypted');
+              assert_equals(event.initDataType, 'cenc');
+              // at this point we do not know if the event is related to audio or video. So check for both expected init data
+              // TODO improve this
+              if (checkInitData(currentData, expectedInitData[0]) || checkInitData(currentData, expectedInitData[1])) {
+                  assert_true(true, true);
```

Comment:
This looks odd. Second parameter to `assert_true` is the description. You could do

```
assert_true( checkInitData(currentData, expectedInitData[0]) || checkInitData(currentData, expectedInitData[1] );
```
and you would not need the `forceTestFailureFromPromise`